firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -116,6 +113,30 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
         setEditing(false, animated: false)
+        unsubscribeFromRedux()
+    }
+
+    // MARK: - Redux
+
+    func subscribeToRedux() {
+        store.dispatch(SearchSettingsAction.searchSettingsDidLoad)
```

Comment:
Should this be dispatching this search settings action, or should this be `store.dispatch(ActiveScreensStateAction.showScreen(.searchSettingsViewController))`? @OrlaM

Reply:
Just double checking, on this, @OrlaM.